faq/index.md

```diff
@@ -12,16 +12,17 @@ Am Ende der Installationsanleitungen gibt es den Punkt
 
 # Linux
 
-## <a id="w-lan"></a>W-LAN unter Ubuntu
+## <a id="w-lan"></a>W-LAN unter Ubuntu/Linux
 
 Um das eduroam-Netz an der TU Dortmund einzurichten, können folgende Optionen verwendet werden:
  - Wi-Fi security: WPA- & WPA2-Enterprise
  - Authentication: Geschütztes EAP (PEAP)
- - Anonymous Identity:  
+ - Anonymous Identity:
```

Comment:
This should be `telesec@tu-dortmund.de` according to https://service.tu-dortmund.de/ubuntu-wlan

Reply:
For me that field is empty, do you have the site cached?

Reply:
The screenshots don't match the text... and the Ubuntu 18 don't match the Ubuntu 12 settings, although there should be no difference between the two for the basic wifi settings of the network... 

I remember needing the anonymous identity